bricks/next-builder/src/data-providers/chunks/doLintStoryboard.ts

```diff
@@ -174,6 +175,7 @@ export function doLintStoryboard(storyboard: Storyboard): StoryboardError[] {
   const tagNameAsTargetRegExp = /^[-\w]+(\\\.[-\w]+)*$/;
   const providerBrickRegExp = /^providers-of-/;
   const customProviderBrickRegExp = /\.provider-/;
+  const injectExp = /^\$\{[a-zA-Z._]*\}$/;
```

Comment:
占位符：

1. 没有要求一定要完整匹配开始与结束
2. 没有要求大括号内一定要以特定字符组成
3. 还有 transform 是否也同时提示？

所以这里改成这样更好：

```suggestion
  const injectExp = /[$@]\{.*?\}/;
```

Reply:
2、3认同。而1的话，如果不限定完整匹配开始与结束，会不会匹配到模板字符串，比如`to: '${APP.homepage}/search'`？

Reply:
2、3认同。而1的话，如果不限定完整匹配开始与结束，会不会匹配到模板字符串，比如`to: '${APP.homepage}/search'`？

Reply:
`'${APP.homepage}/search'` 这个就是占位符哦